2Dgame/Assets/Scripts/PlatformSpawnerScript.cs

```diff
@@ -123,15 +125,44 @@ void choosePlatform()
 
 	/**
 	 * Returns a random x-position to spawn a platform at.
-	 * Makes two platforms in the same position twice in a row
+	 * Makes two platforms in the same position twice
 	 * in a row less likely, though not impossible
 	 */
 	int correctX() {
-		int x = 2 * Random.Range (-3, 4);
+        int x = Random.Range(-6, 7);
```

Comment:
Don't know, but this change doesn't really belong here either way. Leaving it out of this pull request.